python/semantic_kernel/processes/process_message_factory.py

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+from typing import Any
+
+from semantic_kernel.processes.kernel_process.kernel_process_edge import KernelProcessEdge
+from semantic_kernel.processes.process_message import ProcessMessage
+
+
+class ProcessMessageFactory:
```

Comment:
I'm never the biggest fan of the factory pattern in Python, but if there are good reasons...